politeiawww/cmd/politeiawwwcli/commands/newinvoice.go

```diff
@@ -0,0 +1,183 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+
+	"github.com/decred/politeia/politeiad/api/v1/mime"
+	"github.com/decred/politeia/politeiawww/api/cms/v1"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+	"github.com/decred/politeia/util"
+)
+
+// NewInvoiceCmd submits a new invoice.
+type NewInvoiceCmd struct {
+	Args struct {
+		Month       string   `positional-arg-name:"month"`           // Invoice Month
+		Year        string   `positional-arg-name:"year"`            // Invoice Year
+		CSV         string   `positional-arg-name:"csvfile"`         // Invoice CSV file
+		Attachments []string `positional-arg-name:"attachmentfiles"` // Invoice attachment files
+	} `positional-args:"true" optional:"true"`
+	Random bool `long:"random" optional:"true"` // Generate random invoice data
```

Comment:
`politeiawwwcli newinvoice --random` throws the error: `strconv.Atoi: parsing "": invalid syntax`